src/utils/DualScreenHelpers.ts

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
Thinking ahead, I think we should rename this file to `WindowSegmentHelpers` since it could be useful for more than just dual screen scenarios.

Reply:
I like it